### DISCUSSION TITLE
curious bug

### DISCUSSION THREAD

**Discussion by abyrnoradan** [WinEdt-Team/WinEdt-Support#65] [1.1 General Help and Discussion]
I had some problem with xy and babel. I tested the following
\documentclass  {article}
\usepackage[all]{xy}
\begin{document}
$\xymatrix{C& D\ar_{g}[l];[] }$
\end{document}

and got a bug. Compilation enden with a " * "  as when an "end" is missing. I entered " x " and got the 
"missing \begin{document} "

!!!!!!!!!!!!

I tested with  texworks and it works so i deduce that the problem comes from winedt. No idea why ?

**Comment by Alex-Simonic** [comment]
You deduce incorrectly... It depends on what TeX engine you use in WinEdt and TeXWorks (you cannot compare apples and oranges).

This sample will not compile with LuaLaTeX but works fine with XeLaTeX or pdfLaTeX.

With LuaLaTeX you get (copied from WinEdt Console):

```
Command Line:   lualatex.exe --interaction=errorstopmode --synctex=-1 "testxy.tex"
Startup Folder: C:\xxx\Temp

This is LuaHBTeX, Version 1.18.1 (MiKTeX 24.4)
 restricted system commands enabled.
(./testxy.tex
LaTeX2e <2024-06-01> patch level 2
L3 programming layer <2024-08-16>

etc...

! Package xypdf Error: pdfTeX version 1.40.0 or higher is needed for the xypdf
package with PDF output.

See the xypdf package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.3 \begin
        {document}
```

This has nothing to do with WinEdt. XeLaTeX (launched from WinEdt) works:

```
Command Line:   xelatex.exe --interaction=errorstopmode --synctex=-1 "testxy.tex"
Startup Folder: C:\xxx\Temp

This is XeTeX, Version 3.141592653-2.6-0.999996 (MiKTeX 24.4) (preloaded format=xelatex.fmt)
 restricted \write18 enabled.
entering extended mode
(testxy.tex
LaTeX2e <2024-06-01> patch level 2
L3 programming layer <2024-08-16>

etc...

<xymatrix 2x1 54>
[1] (testxy.aux) )
Output written on testxy.pdf (1 page).
SyncTeX written on testxy.synctex.
Transcript written on testxy.log.
```
!!!!!!!!

Now you know :-) You should provide information like that if you report a "bug" with compilation.

**Reply by abyrnoradan** [reply]
I use of course the same "LaTex" in both !

**Reply by abyrnoradan** [reply]
Oups ! 
Sorry winedt, sorry ! 
Of course, I thought that a "TeX" file was saved with a "tex" extension as in old winedt . I did not remark my file was without extension !
But it stay curious. 
Why a file without extension bug in winedt ?

Of course something to do with configuration of winedt but what ?
What happen if the file has no extension or a wrong one and of course is "TeXly" good written?

**Comment by Alex-Simonic** [comment]
Are you a foolish person or just a bot? I have to ask as you seem to assume that crystal globes can be used to reproduce your (double quotes) unreproducible "bug" reports. Sorry, but you have run out of my patience zone. Re-read previous answers or stick with what you can handle. OK with me if anybody else wants to help you... But at least tell them what exactly you did or use some other software. Fair enough?
